platforms/documentation/docs/src/docs/userguide/reference/glossary.adoc

```diff
@@ -15,6 +15,25 @@ Directories are common in the case of inter-project dependencies to avoid the co
 
 == B
 
+[[sub:terminology_build]]
+Build::
+A build is the aggregate of the atomic pieces of work performed by Gradle.
+It is made up of <<sub:terminology_project,projects>> and those projects have a collection of <<sub:terminology_task, tasks>>.
++
+A build usually has an outcome of SUCCESS or FAILURE.
++
+You can run a build using the `gradle` or `gradlew` commands.
+
+[[sub:terminology_build_scan]]
+Build Scan::
+A Build Scan is a service provided by Gradle which is used to identify build issues and build performance.
```

Comment:
💅 consider adding a link to https://gradle.com/develocity/product/build-scan/ or something like that.

Reply:
Added.